api/proto/disperser/disperser.proto

```diff
@@ -97,18 +97,25 @@ message DisperseBlobReply {
 	// The status of the blob associated with the request_id. Will always be PROCESSING.
 	BlobStatus result = 1;
 	// The request ID generated by the disperser.
-	// Once a request is accepted (although not processed), a unique request ID will be
-	// generated.
-	// Two different DisperseBlobRequests (determined by the hash of the DisperseBlobRequest)
-	// will have different IDs, and the same DisperseBlobRequest sent repeatedly at different
-	// times will also have different IDs.
-	// The client should use this ID to query the processing status of the request (via
-	// the GetBlobStatus API).
+	//
+	// Once a request is accepted, and before it is processed, a unique request ID is generated.
+	// request_id = string(blob_key) = (hash(blob), hash(metadata))
+	// where metadata contains a requestedAt timestamp and the requested quorum numbers and their adversarial thresholds.
+	// BlobKey definition: https://github.com/Layr-Labs/eigenda/blob/6b02bf966afa2b9bf2385db8dd01f66f17334e17/disperser/disperser.go#L87
+	// BlobKey computation: https://github.com/Layr-Labs/eigenda/blob/6b02bf966afa2b9bf2385db8dd01f66f17334e17/disperser/common/blobstore/shared_storage.go#L83-L84
+	//
+	// Two different DisperseBlobRequests thus have different IDs, whereas the same DisperseBlobRequest 
```

Comment:
is "whereas" the right conj. here :D

Reply:
good call. rephrased in [c815197](https://github.com/Layr-Labs/eigenda/pull/1360/commits/c8151978111231826263bfc7de4b42ca1ba75458)